.gitignore

```diff
@@ -30,3 +30,5 @@ coverage.*
 *.log
 
 cpma-config.yaml
+
+rice-box.go
```

Comment:
I think we will need this file to be included in source code in order to build this from release-engineering tools.

I doubt we will be successful in getting rel-eng processes to run 'rice embed-go' prior to building CPMA.

Pattern we've followed in past for these kind of things are to check in the generated resources.
It's not ideal, but it's been the path of least pain for integrating with processes.

Reply:
We decided to leave this in .gitignore for now, will revisit during rel-eng work and may end up committing this file.